test.e2e/spec/services.spec.js

```diff
@@ -443,7 +443,8 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
           'deleteById',
           'removeById',
           'count',
-          'prototype$updateAttributes',
+          'prototype$patchAttributes',
+          'updateAttributes',
```

Comment:
[in this test case it manually modifies the string so the test case passes](https://github.com/strongloop/loopback/pull/2316/files#diff-e43576aabfd6aa741af8ac58f1b4a699R627).... 
is it a problem with https://github.com/strongloop/strong-remoting/blob/master/lib/rest-adapter.js has no concept of aliases? which is what [sdk-angular](https://github.com/strongloop/loopback-sdk-angular/blob/bf249964e16c40a94b3f22dd4183e23f6a037b62/lib/services.js#L95) interfaces with
@bajtos thoughts?